### PR TITLE
Evt_Communication Update

### PR DESCRIPTION
**Pull request by fantoms**
* Added Emote
* Added HearEmote 
* Added SoulEmote
* Added HearSoulEmote

### CODE REVIEW

**Comment by fantoms** [comment]
* Added TransientString

**Comment by fantoms** [comment]
* Added PopUpString: // 0x0004

File: aclogview/CM_Communication.cs [hunk 1]
```diff
@@ -176,6 +231,96 @@ public override void contributeToTreeView(TreeView treeView)
...
+            rootNode.Nodes.Add("emoteMessage = " + emoteMessage);
+            treeView.Nodes.Add(rootNode);
+        }
+    }
+
+    public class HearEmote : Message
+    {
+        public PStringChar EmoteMessage;
+        public PStringChar SenderName;
+        public uint SenderID;
```
Comment (tfarley) [inline aclogview/CM_Communication.cs]: Please reorder the declarations and order of nodes in the tree view to match the read order

File: aclogview/CM_Communication.cs [hunk 2]
```diff
@@ -176,6 +231,96 @@ public override void contributeToTreeView(TreeView treeView)
...
+            rootNode.Expand();
+            rootNode.Nodes.Add("EmoteMessage = " + EmoteMessage.m_buffer);
+            treeView.Nodes.Add(rootNode);
+        }
+    }
+    public class HearSoulEmote : Message
+    {
+        public PStringChar EmoteMessage;
+        public PStringChar SenderName;
+        public uint SenderID;
```
Comment (tfarley) [inline aclogview/CM_Communication.cs]: Same here - Please reorder the declarations and order of nodes in the tree view to match the read order

**Review by tfarley — CHANGES_REQUESTED** [review]
Looking good. Just a couple ordering things.

**Comment by fantoms** [comment]
The 2 changes request have been corrected. Thank you for looking over my contributions, once again!

**Comment by fantoms** [comment]
These changes should be ready to merge if you're comfortable with them.

**Pull request merged**